docs/credits.rst

```diff
@@ -268,6 +283,7 @@ Core Package Contributors
 * Nadia Dencheva
 * Nathanial Hendler
 * Nathaniel Starkman
+* Nathaniel Starkman
```

Comment:
Duplicate ?
```suggestion
```

Reply:
I've added this along with a mailmap cleanup